### DISCUSSION TITLE
is there a way to ignore videos with no transcripts when using "get_transcripts"

### DISCUSSION THREAD

**Discussion by ddawsari** [jdepoix/youtube-transcript-api#143] [Q&A] [answered]
Hello!

thanks for the cool project :D 

I have been playing around with it and I came across the "get_transcripts" function which can retrieve multiple transcripts at the same time, however, if any of the videos don't have a transcript, it stops and doesn't return any results except the warning message about the video. am I missing something or could this be an area of improvement? 

Thanks

**Comment by jdepoix** [comment, marked as answer]
Hi @ddawsari,
if you don't want `get_transcripts` to stop on errors you can run `get_transcripts(video_ids, continue_after_error=True)`.

**Reply by ddawsari** [reply]
Thank you !
